content/spaces/cursors.textile

```diff
@@ -40,7 +40,7 @@ The @set()@ method takes the following parameters:
 The following is an example of a member setting their cursor position by adding an event listener to obtain their cursor coordinates and then publishing their position using the @set()@ method:
 
 ```[javascript]
-window.addEventListener('mousemove', ({ clientX, clientY }) => {
+window.addEventListener('mousemove', async ({ clientX, clientY }) => {
   await space.cursors.set({ position: { x: clientX, y: clientY }, data: { color: 'red' } });
```

Comment:
I'm not sure we should present this example like this. The `await` here is correct but not needed. It would only be needed, if in the same function, there would be another operation that requires that the cursor is set and confirmed so. So although this is an async operation we don't need to add async/await keywords here.

Reply:
Ah thanks Dom - I've fixed both of those.